media-video/webcamoid/webcamoid-9999.ebuild

```diff
@@ -50,18 +49,13 @@ RDEPEND="${COMMON_DEPEND}
 "
 
 src_configure() {
-	#Disable git in package source. If not disabled the cmake configure process will show a lot of "fatal not a git repository" errors
```

Comment:
Since this ebuild is using git to download the source from the latest commit this build will not result in "fatal not a git repository" errors